Sources/Web3Wallet/Web3Wallet.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+import Combine
+
+import Auth
+import WalletConnectSign
+
+/// Web3Wallet instance wrapper
+///
+/// ```Swift
+/// let metadata = AppMetadata(
+///     name: "Swift wallet",
+///     description: "wallet",
+///     url: "wallet.connect",
+///     icons:  ["https://my_icon.com/1"]
+/// )
+/// Web3Wallet.configure(metadata: metadata, account: account)
+/// Web3Wallet.instance.getSessions()
+/// ```
+public class Web3Wallet {
+    /// Web3Wallett client instance
+    public static var instance: Web3WalletClient = {
+        guard let config = Web3Wallet.config else {
+            fatalError("Error - you must call Wallet.configure(_:) before accessing the shared instance.")
+        }
+        
+        return Web3WalletClientFactory.create(
+            metadata: Pair.metadata,
+            projectId: Networking.projectId,
+            signerFactory: config.signerFactory,
+            networkingClient: Networking.interactor,
+            pairingRegisterer: Pair.registerer,
+            pairingClient: Pair.instance as! PairingClient
```

Comment:
@llbartekll as I remember we found the way how to avoid force casts for pairing client. Or we decided to leave it like that?

Reply:
We have fixed it for Networking having two vars: `instance` and `interactor` I think we could do the same with pairing